Stockchecker.js

```diff
@@ -84,4 +83,4 @@ function doesntexist(element, name) {
 
 
 
-console.log(doesntexist(product, 'spoon'));
\ No newline at end of file
+console.log(doesntexist(product, 'Table'));
```

Comment:
function totalStockAmount(element) -> function totalStockAmount(products) it's more descriptive

Reply:
the same fix for all the other functions